src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksDtoToPaymentWorksVendorConversionServiceImpl.java

```diff
@@ -131,6 +133,17 @@ private void populateNewVendorBankAccountAttributes(PaymentWorksVendor stgNewVen
             populateNewVendorBankAddressAttributes(stgNewVendor, pmwBankAccountDTO);
         }
     }
+    
+    protected String sanitizeBankAcctBankAccountNumberValue(String pmwBankAccountNumber) {
+        if (StringUtils.isNotBlank(pmwBankAccountNumber) && StringUtils.contains(pmwBankAccountNumber, DASH)) {
```

Comment:
Can this be simplified to just do the StringUtils.contains check?  If the account number has a dash, it can't be blank.

Reply:
simplified